src/assets/Enums.ts

```diff
@@ -174,19 +174,30 @@ export const TEST_TYPES_GROUP2: string[] = [
     "15", "16", "23", "19", "22"
 ];
 
-export const TEST_TYPES_GROUP3_4_5_10: string[] = [
-    "38", "30", "33", "34", "32", "31", "100", "121", "36", "86", "88", "89", "90", "56", "59", "60", "87", "47", "48", "49", "50", "85", "57"
+export const TEST_TYPES_GROUP3_4_8: string[] = [
```

Comment:
These have gone from meaningless to unintelligible and meaningless. If these consts represent logical groupings, can there be comments explaining why these enums exist

Reply:
done